### PR TITLE
Kogito 1.5.0 productization changes

### PR DESCRIPTION
**Pull request by almope**
Signed-off-by: Alberto Morales Perez <almorale@redhat.com>

As this was needed for 1.5.0 productization it has been checked in the backport first: https://github.com/kiegroup/kogito-runtimes/pull/1261

Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [ ] You have read the [contributors guide](CONTRIBUTING.md)
- [x] Your code is properly formatted according to [this configuration](https://github.com/kiegroup/kogito-runtimes/tree/master/kogito-ide-config)
- [x] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [ ] Pull Request title contains the target branch if not targeting master: `[0.9.x] KOGITO-XYZ Subject`
- [ ] Pull Request contains link to the JIRA issue
- [x] Pull Request contains link to any dependent or related Pull Request
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>Pull Request</b>  
  Please add comment: <b>Jenkins retest this</b>
 
* <b>Quarkus LTS checks</b>  
  Please add comment: <b>Jenkins run LTS</b>

* <b>Native checks</b>  
  Please add comment: <b>Jenkins run native</b>

* <b>Full Kogito testing</b> (with cloud images and operator BDD testing)  
  Please add comment: <b>Jenkins run BDD</b>  
  <b>This check should be used only if a big change is done as it takes time to run, need resources and one full BDD tests check can be done at a time ...</b>
</details>

### CODE REVIEW

File: addons/cloudevents/pom.xml [hunk 1]
```diff
@@ -18,36 +18,36 @@
...
+        <groupId>org.kie.kogito</groupId>
+        <artifactId>addons</artifactId>
+        <version>2.0.0-SNAPSHOT</version>
+    </parent>
+    <artifactId>cloudevents</artifactId>
+    <packaging>pom</packaging>
+    <name>Kogito :: Add-Ons :: Cloud Events</name>
 
-  <description>Kogito Cloud Events</description>
+    <description>Kogito Cloud Events</description>
```
Comment (mareknovotny) [inline addons/cloudevents/pom.xml:31]: @almope is this XML formatting correct?

Comment (mareknovotny) [reply addons/cloudevents/pom.xml:31]: i can see all xml files use probably only 2 spaces instead of 4 spaces like here

Comment (almope) [reply addons/cloudevents/pom.xml]: @mareknovotny ok, fixed the formatting. Thanks!

**Review by mareknovotny — APPROVED** [review]

**Comment by mareknovotny** [comment]
jenkins retest this please

**Comment by almope** [comment]
jenkins retest this

**Comment by almope** [comment]
jenkins retest this please

**Comment by almope** [comment]
jenkins retest this

**Pull request merged**